### PR TITLE
Update Legal Strings in Your Consent Screens (EXPOSUREAPP-6030)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Updated the legal strings in both "Your Consent" Screens. 
To validate the new strings use the current version in Figma :)
To validate the translations into the foreign languages, please compare with the texts written in ticket 6387

### CODE REVIEW

**Review by chiljamgossow — COMMENTED** [review]
Looks fine, just missing the new text on the onboarding screen

**Comment by axelherbstreith** [comment]
@chiljamgossow wanted to do that in a separate PR :)

**Review by chiljamgossow — DISMISSED** [review]

**Comment by harambasicluka** [comment]
@axelherbstreith & @chiljamgossow I just pushed the onboarding changes to this branch, also did some minor clean up.

**Comment by AlexanderAlferov** [comment]
Checked current changes in strings and layout - looks good.

**Review by AlexanderAlferov — APPROVED** [review]

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -413,6 +413,16 @@
...
     <!-- YTXT: onboarding(together) - explain application -->
     <string name="onboarding_body_emphasized">"The app logs encounters between individuals by exchanging encrypted, random IDs between their smartphones, whereby no personal data whatsoever is accessed."</string>
+    <!-- YTXT: onboarding(together) - paragraph -->
+    <string name="onboarding_body_1">"More protection for you and for us all. By using the Corona-Warn-App we can break chains of infection much quicker."</string>
+    <!-- YTXT: onboarding(together) - paragraph -->
+    <string name="onboarding_body_2">"Turn your smartphone into a coronavirus warning system. Get an overview of your risk status and find out whether you’ve had close contact with anyone diagnosed with coronavirus in the last 14 days."</string>
+    <!-- YTXT: onboarding(together) - paragraph -->
+    <string name="onboarding_body_3">"Lassen Sie sich Ihre Testergebnisse (PCR-Test oder Antigen-Schnelltest) in der App anzeigen und warnen Sie andere, wenn Sie ein positives Testergebnis erhalten."</string>
```
Comment (janetback) [inline Corona-Warn-App/src/main/res/values/strings.xml:421]: Could you please check <string name="onboarding_body_3" and <string name="onboarding_body_4" - German in values file?

**Review by janetback — CHANGES_REQUESTED** [review]
Hi, could you please check values/stirngs.xml - > is the language mix intended? See also comment.

**Review by janetback — APPROVED** [review]
Hi, just communicated with @harambasicluka - these values will be overwritten with values from English -> am UA approving as is.

**Review by d4rken — APPROVED** [review]

**Pull request merged**